src/middleware/error-handler.ts

```diff
@@ -1,9 +1,19 @@
 import CustomAPIError from '../errors/custom-api-error'
 import { type Request, type Response, type NextFunction, type ErrorRequestHandler } from 'express'
```

Comment:
It is not necessary to add "type" in imports

Reply:
Yes, it's just part of my eslint configuration, including @typescript-eslint/consistent-type-imports and @typescript-eslint/consistent-type-exports, and is placed automatically when npm lint is run. If needed, I can remove these rules😄

Reply:
Please, remove sensitive data and that's ok

Reply:
Done! Sensitive data has been removed, and the database access has been changed. Let me know if there's anything else I should tweak